examples/themes/template-theme.html

```diff
@@ -0,0 +1,204 @@
+          </symbol>
+        </svg>
+
+        <media-controller>
+          <slot name="media" slot="media"></slot>
+          <media-poster-image slot="poster" part="poster" src="{{poster}}" placeholder-src="{{placeholder}}"></media-poster-image>
+          <media-loading-indicator slot="centered-chrome" no-auto-hide></media-loading-indicator>
+
+          <template directive="layout" expression="on-demand extra-small">
+            <media-control-bar>
+              <media-play-button disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-play-button>
+              <media-time-range disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-time-range>
+              <media-fullscreen-button disabled="{{disabledFullscreen}}" aria-disabled="{{disabledFullscreen | string}}">
+                <svg slot="enter" viewBox="0 0 24 24"><use xlink:href="#fullscreen-enter" /></svg>
+              </media-fullscreen-button>
+            </media-control-bar>
+          </template>
+
+          <template directive="layout" expression="on-demand large">
+            <media-control-bar slot="top-chrome">
+              <template directive="if" expression="title">
+                <media-text-display part="top title display">
+                  <b>{{title | string}}</b>
+                </media-text-display>
+              </template>
+            </media-control-bar>
+            <div slot="centered-chrome" class="center-controls">
+              <!-- Here the modifier casts this expression value to a string, syntax is up to us. Can be changed -->
+              <media-play-button part="center play button" disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-play-button>
+            </div>
+            <media-control-bar>
+              <media-play-button disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-play-button>
+              <media-time-range disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-time-range>
+              <media-fullscreen-button disabled="{{disabledFullscreen}}" aria-disabled="{{disabledFullscreen | string}}">
+                <svg slot="enter" viewBox="0 0 24 24"><use xlink:href="#fullscreen-enter" /></svg>
+              </media-fullscreen-button>
+            </media-control-bar>
+          </template>
+
+          <!-- This could be a solution for when you'd like to re-use certain elements across different layouts -->
+          <!-- In this example the play and fullscreen button in the bottom control bar could be the same elements in extra small and large,
+                not working yet because they have to be cached in the template parts processor -->
+          <!-- It's quite verbose but this would be the foundation on which it is build, I could see we introduce non-HTML language
+                in the future but it would translate to what you see below at runtime if we chose template parts -->
```

Comment:
It does feel pretty verbose, but makes sense as a starting point and good to see the inner workings.

The proposals suggest mustache syntax, which has `{{> thing}}` for partials like this, if we should consider that. Though I'm not sure how you define a partial in the first place within a mustache doc. I only see how to do it in a different file.

Reply:
`{{> }}` syntax is totally doable too, to make a difference between escaped and non escaped html